examples/http-1-1-to-2-0-protocol-switch/http_1_1_to_2_0_protocol_switch.bal

```diff
@@ -4,26 +4,16 @@ import ballerina/http;
 http:Client http2serviceClientEP =
         check new ("http://localhost:7090", {httpVersion: "2.0"});
 
-service /http11Service on new http:Listener(9090) {
+service / on new http:Listener(9090) {
```

Comment:
Do we need this `/` here?

```suggestion
service on new http:Listener(9090) {
```

Reply:
@ThisaruGuruge `/` is not needed, but kept it intentionally to show the basePath